index.html

```diff
@@ -1,24 +1,45 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Index</title>
+<title>Mount Sion CBS Secondary School</title>
     
     <link rel="stylesheet" href="style.css">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.css">
-  <link src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.js"></link>
+  <link src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.js">
    <script src="//code.jquery.com/jquery-2.2.1.js"></script>
     
 </head>
  <body>
     
-<ul>
-  <li><a class="active" href="index.html">Home</a></li>
-  <li><a href="about.html">About</a></li>
-  <li><a href="history.html">History</a></li>
-  <li><a href="contact.html">Contact</a></li>
-</ul>
-    
-    
-    
+<div class="ui container">
```

Comment:
The container was only a suggestion, you did not have to add it, but if you are adding it, I would wrap the whole body in it, it will add some nice padding to the sides of the website. Have a look at this [example](https://gist.github.com/ciaranRoche/4d03523e578fef9c3d73b0d3158caebf) to see what I am talking about